src/containers/WalletAdd/WalletAdd.tsx

```diff
@@ -96,6 +96,7 @@ class WalletAdd extends React.Component<Props, State> {
       Contacts.Store.addContact(
         ContactRepository.getContactFromIdentity(newIdentity, {
           unregistered: true,
+          addedAt: Date.now(),
```

Comment:
Why was this added?

Reply:
Its the fix in the develop branch, I will remove the commit.

It shouldn't be inside.
=